common/src/main/java/com/wynntils/features/inventory/MythicBlockerFeature.java

```diff
@@ -18,7 +18,7 @@
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-@ConfigCategory(Category.COMBAT)
+@ConfigCategory(Category.INVENTORY)
 public class MythicBlockerFeature extends Feature {
```

Comment:
I am skeptical, but can possibly be convinced that this is correct. The problem is that "Inventory" is already a too crowded category. And as I suggested, maybe rename "Combat" to "Gameplay"?